Purchases/Public/Purchases.swift

```diff
@@ -1408,13 +1423,6 @@ public extension Purchases {
 }
 
 // MARK: Delegate implementation
-extension Purchases: CustomerInfoManagerDelegate {
-
-    public func customerInfoManagerDidReceiveUpdated(customerInfo: CustomerInfo) {
-        delegate?.purchases?(self, receivedUpdated: customerInfo)
```

Comment:
I think we still need to keep this around, otherwise apps targeting iOS 11-12 won't be able to use the delegate

Reply:
The public delegate is still around :)

Reply:
This is removing `CustomerInfoManagerDelegate` which was `internal`. The method in `PurchasesDelegate` is still there.

Reply:
Did you get a chance to look at this? :)